{"uid":"7c7d02a689708f4d","name":"test_edit_account_field_unknow","fullName":"test.Cuentas.test_edit_account#test_edit_account_field_unknow","historyId":"556de1231d9bca46362a6c04705654b1","time":{"start":1726781018377,"stop":1726781018378,"duration":1},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nTypeError: PayloadAccount.build_payload_add_account() got an unexpected keyword argument 'test'","statusTrace":"setup_edit_account = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, {'assignedUserId': None, 'assignedUserName': None, 'billingAddressCity': None, 'billingAddressCountry': None, ...})\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_edit_account_field_unknow(setup_edit_account):\n        headers, account = setup_edit_account\n>       payload_account = PayloadAccount().build_payload_add_account(name=\"prueba nueva\", billingAddressCity=\"prueba\",\n                                                                     billingAddressCountry=\"prueba\",\n                                                                     billingAddressPostalCode=\"prueba\",\n                                                                     billingAddressState=\"prueba\", test=\"test\",\n                                                                     billingAddressStreet=\"prueba\",\n                                                                     description=\"prueba\", emailAddress=\"prueba@prueba.com\",\n                                                                     industry=\"Architecture\", phoneNumber=\"+15555555555\",\n                                                                     shippingAddressCity=\"prueba\",\n                                                                     shippingAddressCountry=\"prueba\",\n                                                                     shippingAddressPostalCode=\"prueba\",\n                                                                     shippingAddressState=\"prueba\",\n                                                                     shippingAddressStreet=\"prueba\", type=\"Customer\",\n                                                                     website=\"prueba.com\")\nE       TypeError: PayloadAccount.build_payload_add_account() got an unexpected keyword argument 'test'\n\ntest/Cuentas/test_edit_account.py:60: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726780996353,"stop":1726780996353,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_edit_account","time":{"start":1726781018182,"stop":1726781018377,"duration":195},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_edit_account::0","time":{"start":1726781018382,"stop":1726781018555,"duration":173},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Cuentas"},{"name":"suite","value":"test_edit_account"},{"name":"host","value":"fv-az1114-828"},{"name":"thread","value":"2097-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Cuentas.test_edit_account"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"7c7d02a689708f4d.json","parameterValues":[]}